tests/Responses/Expected/taxon/index_response.json

```diff
@@ -0,0 +1,217 @@
+{
+    "page": 1,
+    "limit": 10,
+    "pages": 1,
+    "total": 7,
+    "_links": {
+        "self": {
+            "href": "\/api\/v1\/taxons\/?page=1&limit=10"
+        },
+        "first": {
+            "href": "\/api\/v1\/taxons\/?page=1&limit=10"
+        },
+        "last": {
+            "href": "\/api\/v1\/taxons\/?page=1&limit=10"
+        }
+    },
+    "_embedded": {
+        "items": [
+            {
+                "id": @integer@,
+                "code": "category",
+                "position": 0,
+                "translations": {
+                    "en": {
+                        "locale": "en"
+                    }
+                },
+                "images": [],
+                "_links": {
+                    "self": {
+                        "href": "@string@"
+                    }
+                }
+            },
+            {
+                "id": @integer@,
+                "code": "mugs",
+                "root": {
+                    "_links": {
+                        "self": {
+                            "href": "@string@"
+                        }
+                    }
+                },
+                "parent": {
+                    "_links": {
+                        "self": {
+                            "href": "@string@"
+                        }
+                    }
+                },
+                "position": 0,
+                "translations": {
+                    "en": {
+                        "locale": "en"
+                    }
+                },
+                "images": [],
+                "_links": {
+                    "self": {
+                        "href": "@string@"
+                    }
+                }
+            },
+            {
+                "id": @integer@,
+                "code": "stickers",
+                "root": {
+                    "_links": {
+                        "self": {
+                            "href": "@string@"
+                        }
+                    }
+                },
+                "parent": {
+                    "_links": {
+                        "self": {
+                            "href": "@string@"
+                        }
+                    }
+                },
+                "position": 1,
+                "translations": {
+                    "en": {
+                        "locale": "en"
+                    }
+                },
+                "images": [],
+                "_links": {
+                    "self": {
+                        "href": "@string@"
+                    }
+                }
+            },
+            {
+                "id": @integer@,
+                "code": "books",
+                "root": {
+                    "_links": {
+                        "self": {
+                            "href": "@string@"
+                        }
+                    }
+                },
+                "parent": {
+                    "_links": {
```

Comment:
Did we decide to skip rest of parent information? we should have here `id` and `code` I suppose